lldb/test/Shell/Process/elf-core/aarch64-no-NT_ARM_TLS.yaml

```diff
@@ -0,0 +1,34 @@
+# REQUIRES: aarch64
```

Comment:
Is this requiring the host to be aarch64 or that the aarch64 llvm target is enabled, we should only need the latter for this.

Reply:
It seems that the test does not need either to work, so I removed the requirement.